examples/petsc/bp1.c

```diff
@@ -376,21 +340,21 @@ int main(int argc, char **argv) {
   CeedQFunctionAddInput(qf_setup, "dx", 3, CEED_EVAL_GRAD);
   CeedQFunctionAddInput(qf_setup, "weight", 1, CEED_EVAL_WEIGHT);
   CeedQFunctionAddOutput(qf_setup, "rho", 1, CEED_EVAL_NONE);
-  CeedQFunctionAddOutput(qf_setup, "target", 1, CEED_EVAL_NONE);
```

Comment:
Renamed output "target" to "true_soln", added output "rhs"

Reply:
I'd later like to change the vector to actually be called 'true_soln' as well, but I was going to wait until after we got everything running.